src/uiLocation.ts

```diff
@@ -157,10 +149,6 @@ class UiLocation {
             currentBranch: initializationData.currentBranch,
         });
 
-        this.api = (payload:ApiRequestProps)=> dispatchPostRobotRequest(postRobot, payload);
-
-        this.createAdapter = createSDKAdapter(postRobot);
```

Comment:
why have we removed this ? @Amitkanswal

Reply:
code is been refactor.  So position is been moved